# Task1

# First notebook

### Цей ноутбук містить завдання з лабораторки

In [1]:
print("Hello, Jupyter!")

Hello, Jupyter!


# Task2

## Результат розрахунків суми та різниці

In [4]:
a = 15
b = 7
print(a + b)
print(a - b)

22
8


In [ ]:
a = 15
b = 7
print(a * b)
print(a / b)

# Task3

## Звіт із продажів

### У цьому розділі ми рахуємо підсумки за простими даними

In [7]:
sales = [120, 135, 100, 150, 90, 200, 170]
total = sum(sales)
average = total / len(sales)
maximum = max(sales)
minimum = min(sales)

In [8]:
print("Total: ", total, "\nAverage: ", average, "\nMax: ", maximum, "\nMin: ", minimum)

Total:  965 
Average:  137.85714285714286 
Max:  200 
Min:  90


| Загальна сума | Середнє | Максимум | Мінімум |
|---------------|---------|----------|---------|
|       965     |  137,86 |    200   |    90   |

# Task4

## Оцінки студентів

##### Оцінки студентів та підрахування всіх операцій

In [10]:
students = [("Alex", 85), ("Maria", 92), ("John", 74), ("Olga", 60), ("Max", 48)]
scores = []
for st in students:
    scores.append(st[1])

average = sum(scores) / len(scores)
maximum = max(scores)
minimum = min(scores)
low_scores = [st for st in students if st[1] < 60]

print("Середній бал:", average)
print("Максимум:", maximum)
print("Мінімум:", minimum)
print("Нижче 60:", low_scores)

for name, score in students:
    print(name, "|", "*" * (score // 10))

Середній бал: 71.8
Максимум: 92
Мінімум: 48
Нижче 60: [('Max', 48)]
Alex | ********
Maria | *********
John | *******
Olga | ******
Max | ****


| Середній бал | Максимальний бал | Мінімальний бал |
|--------------|------------------|-----------------|
|    71,8      |        92        |        48       |